gschem/src/gschem_binding_integer.c

```diff
@@ -132,8 +132,8 @@ class_init (GschemBindingIntegerClass *klass)
 {
   GObjectClass *object_class = G_OBJECT_CLASS (klass);
 
-  klass->parent_class.update_model = update_model;
-  klass->parent_class.update_widget = update_widget;
```

Comment:
Aren't these functions already of the correct type?  It looks like they are already correct (taking `GchemBinding*` as only argument and returning `gboolean`).  What warning did you get?

https://github.com/vzh/lepton-eda/blob/2036a7df217c2faf199129b9d17c9cdb472bbc23/gschem/src/gschem_binding_integer.c#L68